### PR TITLE
ID - Story/Episode 5 Update (As of 17/02/19)

### PR DESCRIPTION
**Pull request by Aethernelle**
-Translated all files of Story/Episode 5 in ID branch according to EN branch.
-Some raw files that are already translated in EN need to be added into ID branch.

### CODE REVIEW

File: Story/Episode 5/st_051080.csv [hunk 1]
```diff
@@ -1,55 +1,55 @@
...
+st_051080_0260#0,"""2. Apakah kau bisa membuktikan itu?"""
+st_051080_0270#0,"""Pernyataan itu tadi?<br>Hmm, bagaimana supaya kalian bisa percaya aku..."""
+st_051080_0280#0,"""Hmm bukti, bukti... Apakah ada sesuatu... Oh iya!"""
+st_051080_0290#0,"""Ah, betul! Aku akan menunjukkan trik keren!"""
+st_051080_0300#0,"""Lihat dengan seksama.<br>Tiga, dua, satu..."""
+st_051080_0310#0,"""Eh? Kapan kamu...<br>Tadi itu bukan teleportasi?"""
+st_051080_0320#0,"""Sensasi itu... <%me> dan aku tahu sensasi itu."""
+st_051080_0330#0,"""Itu teknik yang sama dengan yang digunakan Luther untuk menghentikan waktu, kan?"""
+st_051080_0340#0,"""Iya. Dia berhasil menjiplak teknik itu<br>setelah melakukan beberapa eksperimen terhadapku."""
+st_051080_0350#0,"""Oh iya, Su-kun akan marah jika kalian mengklaim bahwa itu adalah Technique, dan bilang<br>\"Jangan bercanda. Ini sihir, bukan Technique.\""""
```
Comment (alama) [inline Story/Episode 5/st_051080.csv]: Missing or stray quote in line 36 in st_051080.csv

Comment (Aethernelle) [reply Story/Episode 5/st_051080.csv]: Yeah, I think I missed a quote line, let me change it quickly. Thanks!

Comment (alama) [reply Story/Episode 5/st_051080.csv]: You KNOW you do not need to close the PR, you can add a commit to fix it, OR your can rewrite your git history to include the change `git rebase -i upstream/ID`

You may want to link CircleCI and TravisCI to your fork so you fix the issues before making a PR

Comment (Aethernelle) [reply Story/Episode 5/st_051080.csv]: I see, thanks for the advice!

Comment (Aethernelle) [reply Story/Episode 5/st_051080.csv]: Seems like after reopening the Pull Request, TravisCI and CircleCI doesn't perform any check on other Commits. Is this alright?

Comment (alama) [reply Story/Episode 5/st_051080.csv]: Yea, the build checks are done when new commits are push on the source branch

**Comment by acffordyce973** [comment]
This is the last merge request we will accept for branches without a maintainer, and any merge requests going forward will be closed. If you'd like to be the maintainer of a branch then contact us at our Discord.

**Pull request merged**